python/ray/serve/storage/kv_storage_base.py

```diff
@@ -0,0 +1,53 @@
+import abc
```

Comment:
nit: can you rename this `kv_store_base.py` to match the other filename (`kv_store.py`)?